osu.Game.Tournament/Screens/SetupScreen.cs

```diff
@@ -108,18 +108,22 @@ private void reload()
                     Items = rulesets.AvailableRulesets,
                     Current = LadderInfo.Ruleset,
                 },
-                resolution = new ActionableInfo
+                resolution = new ActionableInfoWithNumberBox
                 {
                     Label = "Stream area resolution",
-                    ButtonText = "Set to 1080p",
-                    Action = () =>
+                    ButtonText = "Set size",
```

Comment:
Should probably say

```suggestion
                    ButtonText = "Set height",
```

because currently it's unclear which dimension will be set.